velodrome/README.md

```diff
@@ -58,3 +58,50 @@ time.
 
 As an example, the token counter measures the usage of our github-tokens, and
 has a new value every hour. We can push the new value to InfluxDB.
+
+Adding a new project
+====================
+
+Adding a new project is as simple as adding it to [config.yaml](). Typically,
+add the name of your project, the list of repositories. Don't worry about the
+`public-ip` field as the IP will be created later. You can also leave prometheus
+configuration if you don't need it initially.
+
+There are new project specific deployments necessary, and they are
+described [#new-project-deployments](below).
+
+Deployment
+==========
+
+Update/Create deployments
+-------------------------
+
+[config.py]() will generate all the deployments file for you. It reads the
+configuration in [config.yaml]() to generate deployments for each projects and/or
+repositories with proper labels. You can then use `kubectl` labels help you
+select what you want to do exactly, for example:
+
+```
+./config.py # Generates the configuration and prints it on stdout
+./config.py | kubectl apply -f - # Creates/Updates everything
+./config.py | kubectl delete -f - # Deletes everything
+./config.py | kubectl apply -f - -l project=kubernetes # Only creates/updates kubernetes
+./config.py | kubectl apply -f - -l app=fetcher # Only creates/updates fetcher
+```
+
+First time deployments
+----------------------
+
+- Make sure you create the secrets for SQL Proxy, by
+  reading [mysql#set-up-google-cloud-sql-proxy]()
+- Create the certificates secret:
+
+```
+curl -O https://curl.haxx.se/ca/cacert.pem
+kubectl create secret generic certificates --from-file=ca-certificates.crt=cacert.pem
```

Comment:
I agree, there is enough set-up already that it's really un-necessary to add more ... That's kind of orthogonal though, I'll fix it seperately.